nix/m1-support/kernel/config

```diff
@@ -48,7 +48,8 @@ CONFIG_CRASH_DUMP=y
 CONFIG_XEN=y
 # CONFIG_ARM64_PTR_AUTH_KERNEL is not set
 CONFIG_RANDOMIZE_BASE=y
-CONFIG_HIBERNATION=y
+CONFIG_SUSPEND=n
+CONFIG_HIBERNATION=n
```

Comment:
I'm unsure how these changes relate to the governor change. Are sleep and hibernation broken in the current kernel? I'd prefer that they be a separate commit if so.